plugins/MantisCoreFormatting/files/syntax-highlighting/monospace.css

```diff
@@ -0,0 +1,17 @@
+/*
+ * Under "Config > User | Font Family" it is possible to select a font dedicated
+ * for an user.
+ *
+ * The result is an inline style that applies this font to all elements.
+ * Including the <span> elements which wraps the language tokens, within
+ * the <code> element.
+ *
+ * <style>
+ *  * { font-family: "Open Sans" }
+ * </style>
+ *
+ * This rule reapplies/forces the monospace font-family to them.
```

Comment:
I think it would be useful to reference the source of this reapply, i.e. where the *font-family* rule was copied from.

Reply:
The font family for `<code>` elements are originally defined in the bootstrap.css and is overwritten by the inline-style.

https://github.com/mantisbt/mantisbt/blob/a7c9660fd0b5cfaea8bdeef7ba4b2e83e28947f2/css/bootstrap-3.4.1.css#L1529-L1534

Reply:
Sorry for the confusion. I was completely wrong, forget it. 

> Is it worth creating an extra CSS file for just a single rule, wouldn't it be better to move it into markdown.css?

It is only relevant if syntax highlighting is active. No syntax highlighting, no unused rule in markdown.css